app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/confirmation_of_submission.scala.html

```diff
@@ -65,6 +65,6 @@ <h1 class="govuk-panel__title">@title</h1>
 
         <p class="govuk-body govuk-!-margin-bottom-9"><a href="@{routes.LandingPageController.showLandingPage().url}" class="govuk-link">@messages(s"$key.what-next.link.1.start-another-claim")</a></p>
 
-        @paragraph(Html(messages(s"exit-survey.link-text", viewConfig.serviceFeedBackUrl)), Some("govuk-body"))
+        @paragraph(Html(messages(s"exit-survey.link-text", viewConfig.ggSignOut)), Some("govuk-body"))
```

Comment:
Do we need this change?

Reply:
AC1: Then I should be signed out of the Reimbursements service

And taken to the survey page